src/legacy/core_plugins/visualizations/public/np_ready/public/legacy/build_pipeline.ts

```diff
@@ -21,8 +21,8 @@ import { cloneDeep, get } from 'lodash';
 import moment from 'moment';
 import { SerializedFieldFormat } from '../../../../../../../plugins/expressions/public';
 import { fieldFormats, ISearchSource } from '../../../../../../../plugins/data/public';
-import { IAggConfig, setBounds, isDateHistogramBucketAggConfig } from '../../../legacy_imports';
 import { Vis, VisParams } from '../types';
+import { IAggConfig, isDateHistogramBucketAggConfig, setBounds } from '../../../../../data/public';
```

Comment:
setBounds is not `static`, but I decided to leave it as this because it is properly addressed by https://github.com/elastic/kibana/pull/58805